src/Nethermind/Nethermind.JsonRpc/Modules/Eth/GasPrice/EthGasPriceConstants.cs

```diff
@@ -13,7 +13,7 @@ public static class EthGasPriceConstants
         public const int DefaultBlocksLimitMaxPriorityFeePerGas = 100; //Limit for how many blocks we check txs in to add to sortedTxList in case of calculating max priority fee per gas
         public const int TxLimitFromABlock = 3; //Maximum number of tx we can add to sortedTxList from one block
         public const int DefaultIgnoreUnder = 2; //Effective Gas Prices under this are ignored
-        public static readonly UInt256 MaxGasPrice = 500.GWei(); //Maximum gas price we can return
+        public static readonly UInt256 MaxGasPrice = 50.GWei(); //Maximum gas price we can return
         public static readonly UInt256 FallbackMaxPriorityFeePerGas = 200.GWei();
```

Comment:
is this ok to keep this 200 gwei?

Reply:
@leonprou as this is pre EIP-1559 the priority fee is not a concern